Commands/Configuration/MailingConfiguration.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Commands.Configuration;
+public class MailingConfiguration
+{
+    public const string SectionName = "Mailing";
+
+    [Required]
+    public MailServerConfiguration MailServer { get; set; } = default!;
+
+    [Required]
+    public SenderConfiguration Sender { get; set; } = default!;
+
+    public class MailServerConfiguration
+    {
+        [Required(AllowEmptyStrings = false)]
+        public string Host { get; set; } = default!;
+
+        [Range(1, 63535)]
+
+        public int Port { get; set; }
+
+        public string? Username { get; set; }
+
+        public string? Password { get; set; }
+
+        public bool RequiresAuthentication => !string.IsNullOrWhiteSpace(Username) && !string.IsNullOrWhiteSpace(Password);
```

Comment:
This doesn't imply that the server requires authentication, it implies you have credentials.